models/question.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+
+const Option = require('./option');
```

Comment:
import the options schema, not the model. Or define it in this file since its small.